content/en/docs/languages/_index.md

```diff
@@ -27,6 +27,13 @@ follows:
 
 {{% telemetry-support-table " " %}}
 
+{{% alert title="Note" color="warning" %}} Even though the API/SDK status is
+stable, changes to underlying [semantic conventions][SemConv] can interrupt your
```

Comment:
```suggestion
stable, changes to underlying [semantic conventions][sem-conv] can interrupt your
```
Not sure why this is not rendering in hugo (see preview: https://deploy-preview-3995--opentelemetry.netlify.app/docs/languages/), maybe those tags have to be lower case, @chalin ?